stage/services/bastion/main.tf

```diff
@@ -0,0 +1,68 @@
+resource "aws_instance" "bastion-stage" {
+    ami = "${var.ami["eu-west-1"]}"
+    instance_type = "t2.micro"
+    vpc_security_group_ids = ["${aws_security_group.bastion-stage.id}"]
```

Comment:
Do you want to create a new security group, or reuse the existing one. In the latter case you would reference the group as `"${data.aws_security_group.bastion.id}" and then have the following in `input.tf`:

```
data "aws_security_group" "bastion" {
  id = "${var.security_group_id}"
}
```

You would then also need to define the variable in `var.tf` and provide the variable (something like **sg-1234**) in the Terraform Enterprise UI.

At this point we are not using a different VPC configuration (mostly subnets and security groups) for the stage environment or even use a different VPC, but it might be worth doing.